src/Tools/Test Framework/Test Runner/src/DataDrivenTest/DataInputs/TestInputGroups.Page.al

```diff
@@ -0,0 +1,81 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.TestRunner;
+
+page 130462 "Test Input Groups"
+{
+    PageType = List;
+    ApplicationArea = All;
+    UsageCategory = Administration;
+    SourceTable = "Test Input Group";
+    CardPageId = "Test Input";
+    Caption = 'Test Inputs';
+    Editable = false;
+    DeleteAllowed = true;
+    InsertAllowed = false;
+    ModifyAllowed = false;
+
+    layout
+    {
+        area(Content)
+        {
+            repeater(Main)
+            {
+                field(Code; Rec.Code)
+                {
+                    ApplicationArea = All;
```

Comment:
It's not necessary to specify the same ApplicationArea on field level if it already exists on page level.
Applies also to all other new page objects.